src/styles/card.css

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+  
+.card_img {
+  width: 160px;
```

Comment:
why do you hardcode width many times use paddings instead

Reply:
without width and heith picture imac size 627x522